java/code_to_gast/java_helpers.py

```diff
@@ -25,3 +25,23 @@ def array_to_gast(node):
 
     gast["elements"] = gast_list
     return gast
+
+
+def bin_op_to_gast(node):
+    """
+    Binops to gast for java 
+    """
+    gast = {"type": "binOp"}
+    gast["left"] = java_router.node_to_gast(node.operandl.member)
+    gast["op"] = node.operator
+    gast["right"] = java_router.node_to_gast(node.operandr)
+    return gast
+
+
+def int_to_gast(node):
+    """
+    Handles int to gast for positive and negative whole numbers
+    """
+    if len(node.prefix_operators) > 0 and node.prefix_operators[0] == "-":
```

Comment:
Does the len(node.prefix_operators) > 0 do anything? In python, both objects and arrays have length so you might need to make sure it is an array as well since that is your intended purpose.
```
d = {'a':1, 'b':2, 'c':3}
print(len(d)) # ==> 3
```